deploy-manage/uninstall/delete-a-cloud-deployment.md

```diff
@@ -1,35 +1,93 @@
-% - [ ] ./raw-migrated-files/cloud/cloud/ec-billing-stop.html
-% - [ ] ./raw-migrated-files/cloud/cloud/ec-delete-deployment.md
-% - [ ] ./raw-migrated-files/docs-content/serverless/general-billing-stop-project.html
-% - [ ] ./raw-migrated-files/cloud/cloud-enterprise/ece-restore-deployment.md
-%      Notes: if you need to restore a deleted deployment ...
+To delete an {{ech}} deployment, 
 
-⚠️ **This page is a work in progress.** ⚠️
+1. Log in to the [{{ecloud}} Console](https://cloud.elastic.co?page=docs&placement=docs-body).
+2. On the deployment overview page, select **Actions**, then select **Delete deployment** and confirm the deletion.
 
-The documentation team is working to combine content pulled from the following pages:
+When you delete your deployment, billing stops immediately rounding up to the nearest hour.
 
-* [/raw-migrated-files/cloud/cloud-heroku/ech-delete-deployment.md](/raw-migrated-files/cloud/cloud-heroku/ech-delete-deployment.md)
-* [/raw-migrated-files/cloud/cloud-enterprise/ece-delete-deployment.md](/raw-migrated-files/cloud/cloud-enterprise/ece-delete-deployment.md)
-* [/raw-migrated-files/cloud/cloud-enterprise/ece-terminate-deployment.md](/raw-migrated-files/cloud/cloud-enterprise/ece-terminate-deployment.md)
-* [/raw-migrated-files/cloud/cloud/ec-delete-deployment.md](/raw-migrated-files/cloud/cloud/ec-delete-deployment.md)
-* [/raw-migrated-files/cloud/cloud-enterprise/ece-restore-deployment.md](/raw-migrated-files/cloud/cloud-enterprise/ece-restore-deployment.md)
\ No newline at end of file
+:::{{warning}}
+When deployments are deleted, we erase all data on disk, including snapshots. Snapshots are retained for very a limited amount of time post deletion and we cannot guarantee that deleted deployments can be restored from snapshots for this reason. If you accidentally delete a deployment, please contact support as soon as possible to increase the likelihood of restoring your deployment.
+:::
+
+:::{{tip}}
+If you want to keep the snapshot for future purposes even after the deployment deletion, you should [use a custom snapshot repository](/deploy-manage/tools/snapshot-and-restore/elastic-cloud-hosted.md).
+:::
+
+Billing restarts as soon as the deployment is restored.
+
+## Serverless
+
+To delete a severless project,
+
+1. Log in to the [{{ecloud}} Console](https://cloud.elastic.co?page=docs&placement=docs-body).
+2. Find your project on the home page in the **Serverless Projects** card and select **Manage** to access it directly. Or, select **Serverless Projects** to go to the projects page to view all of your projects.
+3. Select **Actions**, then select **Delete project** and confirm the deletion.
+
+:::{warning}
+All data is lost. Billing for usage is by the hour and any outstanding charges for usage before you deleted the project will still appear on your next bill.
+:::
+
+## {{ece}}
+
+:::::{{tab-set}}
+::::{{tab-item}} Delete a deployment
```

Comment:
I think tabs should be used to show parallel processes with different contexts, not different processes. I'd prefer to see these as H3s.